src/EnergyPlus/OutdoorAirUnit.cc

```diff
@@ -1211,372 +1215,51 @@ namespace OutdoorAirUnit {
-										rho = GetDensityGlycol( PlantLoop( OutAirUnit( OAUnitNum ).OAEquip( CompNum ).LoopNum ).FluidName, 5.0, PlantLoop( OutAirUnit( OAUnitNum ).OAEquip( CompNum ).LoopNum ).FluidIndex, RoutineName );
-
-										Cp = GetSpecificHeatGlycol( PlantLoop( OutAirUnit( OAUnitNum ).OAEquip( CompNum ).LoopNum ).FluidName, 5.0, PlantLoop( OutAirUnit( OAUnitNum ).OAEquip( CompNum ).LoopNum ).FluidIndex, RoutineName );
-
-										MaxVolWaterFlowDes = DesCoilLoad / ( PlantSizData( PltSizCoolNum ).DeltaT * Cp * rho );
-									} else {
-										MaxVolWaterFlowDes = 0.0;
-									}
-								} else {
-									ShowSevereError( "Autosizing of water flow requires a Sizing:Zone object or a cooling loop Sizing:Plant object" );
-									ShowContinueError( "Occurs in ZoneHVAC:OutdoorAirUnit Object=" + OutAirUnit( OAUnitNum ).Name );
-									ErrorsFound = true;
-								}
-							}
-							if ( IsAutoSize ) {
-								OutAirUnit( OAUnitNum ).OAEquip( CompNum ).MaxVolWaterFlow = MaxVolWaterFlowDes;
-								ReportSizingOutput( "ZoneHVAC:OutdoorAirUnit", OutAirUnit( OAUnitNum ).Name, "Design Size Maximum Cold Water Flow [m3/s]", MaxVolWaterFlowDes );
-							} else {
-								if ( OutAirUnit( OAUnitNum ).OAEquip( CompNum ).MaxVolWaterFlow > 0.0 && MaxVolWaterFlowDes > 0.0 ) {
-									MaxVolWaterFlowUser = OutAirUnit( OAUnitNum ).OAEquip( CompNum ).MaxVolWaterFlow;
-									ReportSizingOutput( "ZoneHVAC:OutdoorAirUnit", OutAirUnit( OAUnitNum ).Name, "Design Size Maximum Cold Water Flow [m3/s]", MaxVolWaterFlowDes, "User-Specified Maximum Cold Water Flow [m3/s]", MaxVolWaterFlowUser );
-									if ( DisplayExtraWarnings ) {
-										if ( ( std::abs( MaxVolWaterFlowDes - MaxVolWaterFlowUser ) / MaxVolWaterFlowUser ) > AutoVsHardSizingThreshold ) {
-											ShowMessage( "SizeOutdoorAirUnit: Potential issue with equipment sizing for ZoneHVAC:OutdoorAirUnit " + OutAirUnit( OAUnitNum ).Name );
-											ShowContinueError( "User-Specified Maximum Cold Water Flow of " + RoundSigDigits( MaxVolWaterFlowUser, 5 ) + " [m3/s]" );
-											ShowContinueError( "differs from Design Size Maximum Cold Water Flow of " + RoundSigDigits( MaxVolWaterFlowDes, 5 ) + " [m3/s]" );
-											ShowContinueError( "This may, or may not, indicate mismatched component sizes." );
-											ShowContinueError( "Verify that the value entered is intended and is consistent with other components." );
-										}
-									}
-								}
-							}
-						}
-					}
-					SetCoilDesFlow( OutAirUnit( OAUnitNum ).OAEquip( CompNum ).ComponentType, OutAirUnit( OAUnitNum ).OAEquip( CompNum ).ComponentName, OutAirUnit( OAUnitNum ).OutAirVolFlow, ErrorsFound );
+		ZoneEqSizing( CurZoneEqNum ).CoolingAirFlow = true;
+		ZoneEqSizing( CurZoneEqNum ).HeatingAirFlow = true;
+		ZoneEqSizing( CurZoneEqNum ).CoolingAirVolFlow = OutAirUnit( OAUnitNum ).OutAirVolFlow;
+		ZoneEqSizing( CurZoneEqNum ).HeatingAirVolFlow = OutAirUnit( OAUnitNum ).OutAirVolFlow;
```

Comment:
save OA unit flow data for use in child components. tried using the DesignSizeFromParent flag but that didn't work.